docs/root/version_history/current.rst

```diff
@@ -88,7 +88,7 @@ New Features
 * http: added support for `Envoy::ScopeTrackedObject` for HTTP/1 dispatching. Crashes while inside the dispatching loop should dump debug information.
 * http: added support for :ref:`preconnecting <envoy_v3_api_msg_config.cluster.v3.Cluster.PreconnectPolicy>`. Preconnecting is off by default, but recommended for clusters serving latency-sensitive traffic, especially if using HTTP/1.1.
 * http: change frame flood and abuse checks to the upstream HTTP/2 codec to ON by default. It can be disabled by setting the `envoy.reloadable_features.upstream_http2_flood_checks` runtime key to false.
-* original_dst*: added support for :ref:`Original Destination <config_listener_filters_original_dst_filter>` on Windows. With this change Envoy can be used as sidecar proxy on Windows.
+* original_dst*: added support for :ref:`Original Destination <config_listener_filters_original_dst_filter>` on Windows. This enables the use of Envoy as a sidecar proxy on Windows.
```

Comment:
`* original_dst: ....`. No `*` before `:`

Reply:
And the link target should be `<config_listener_filters_original_dst>`

Reply:
fixed, see https://storage.googleapis.com/envoy-pr/2fa1526/docs/version_history/current.html

The docs generation does not work on windows so for doc changes I usually test them via the CI